addon/components/modal-outlet.ts

```diff
@@ -0,0 +1,74 @@
+import Component from '@ember/component';
+// @ts-ignore: Ignore import of compiled template
+import template from '../templates/components/modal-outlet';
+import { inject as service } from '@ember-decorators/service';
+import { action, computed } from '@ember-decorators/object';
+import { tagName, layout } from '@ember-decorators/component';
+import Modal from 'gavant-ember-modals/services/modal';
+import { getWithDefault } from '@ember/object';
+import { assert } from '@ember/debug';
+import { isNone } from '@ember/utils';
+
+@tagName('')
+@layout(template)
+export default class ModalOutlet extends Component {
+    @service modal!: Modal;
+    name: string = 'application';
+
+    @computed('modal.current.{path,outlet,config}')
+    get currentData() {
+        if (this.modal.current) {
+            const path = this.modal.current.path;
+            const config = this.modal.current.config || {};
+            const outlet = this.modal.current.outlet;
+            const actions = getWithDefault(config, 'actions', {});
+
+            if (path) {
+                if (outlet && outlet === this.name) {
+                    return Object.assign({ name: path }, config, actions);
+                }
+            }
+        }
+
+        return null;
+    }
+
+    @computed('currentData')
+    get currentCmp() {
+        const data = this.get('currentData');
```

Comment:
currentData is a regular getter so you should just be able to do `this.currentData` w/ no `get()`s